UDPServer/SiceUDPServer.cs

```diff
@@ -85,5 +84,47 @@ public void Dispose()
         {
             client.Dispose();
         }
+
+        public void Handle(ServerCommand command)
+        {
+            if(command.Command)
+            {
+                ConnectionIP = command.ConnectionIP;
+                ConnectionPort = command.ConnectionPort;
+                this.Connect();
+            }
+            else
+            {
+                this.Disconnect();
+            }
+        }
+
+        public class ServerCommand
+        {
+            public ServerCommand(bool command, string IP, string port)
+            {
+                this.Command = command;
+                if(IPAddress.TryParse(IP, out IPAddress parseIP))
+                {
+                    ConnectionIP = parseIP;
+                }
+                if(int.TryParse(port, out int parsePort))
+                {
+                    ConnectionPort = parsePort;
+                }
+            }
+            public bool Command { get; set; }
+            public IPAddress ConnectionIP { get; set; }
+            public int ConnectionPort { get; set; }
+        }
+
+        public class ServerStatus
+        {
+            public ServerStatus(bool status)
+            {
+                this.Status = status;
+            }
+            public bool Status { get; set; }
```

Comment:
Same as above.

If the status **must** be set in the constructor, then the property should be marked as `private set`

Reply:
Changed